src/components/PrivacyAgent.js

```diff
@@ -39,20 +43,28 @@ class PrivacyAgent { // eslint-disable-line no-unused-vars
                 <p><strong data-i18n="privacy-agent-warning">Anyone that can see your Recovery Words can steal all your funds!</strong></p>
             </div>
             <div class="grow"></div>
-            <a href="#continue"><button data-i18n="privacy-agent-ok">OK, all good</button></a>
+            <button data-i18n="privacy-agent-ok">OK, all good</button>
         `;
-        this.$ok = el.querySelector('a');
-        if (!this.$ok) {
+        /* eslint-enable max-len */
+
+        const $ok = element.querySelector('button');
+        if (!$ok) {
```

Comment:
Just cast to HTMLButtonElement. We know it's there.

Reply:
@svub like this:
```
/** @type {HTMLButtonElement} /*
const $ok = (element.querySelector('button'));
```